ansibox

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+export ANSIBLE_CONFIG=.ansible.cfg
+":" //#explained here sambal.org/?p=1014; exec ansible-playbook -i hosts .playbook.yaml "$@"
```

Comment:
I read this, and I still don't understand it :) Can we use this to run multiple playbooks?